### PR TITLE
Create de_de.lang

### PR DESCRIPTION
**Pull request by ghost**
Translation to German

### CODE REVIEW

File: src/main/resources/assets/rebornstorage/lang/de_de.lang [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+#Fluid Storage Parts
+item.rebornstorage.storagepart.16384k_fluid.name=16384k Flüssigspeicherkern
+item.rebornstorage.storagepart.32768k_fluid.name=32768k Flüssigspeicherkern
+item.rebornstorage.storagepart.131m_fluid.name=131m Flüssigspeicherkern
+item.rebornstorage.storagepart.524m_fluid.name=524m Flüssigspeicherkern
+
+#Blocks
+tile.rebornstorage.multicrafter.heat.name=Wärmeableiter
+tile.rebornstorage.multicrafter.frame.name=Handwerk-Gerüst
```
Comment (Vexatos) [inline src/main/resources/assets/rebornstorage/lang/de_de.lang]: This word really doesn't match the original meaning.

File: src/main/resources/assets/rebornstorage/lang/de_de.lang [hunk 2]
```diff
@@ -0,0 +1,32 @@
+itemGroup.rebornstorage=Reborn Storage
+
+#Items Disks
+item.rebornstorage.storagecell.256k.name=256k Speicherzelle
```
Comment (Vexatos) [inline src/main/resources/assets/rebornstorage/lang/de_de.lang:4]: Not sure, shouldn't all these be connected with hyphens?

**Review by Vexatos — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/rebornstorage/lang/de_de.lang (hunk 2 above)
Comment (ghost) [reply src/main/resources/assets/rebornstorage/lang/de_de.lang:4]: The translation of refined storage also use "Speicherzelle" as far as I know.

Comment (Vexatos) [reply src/main/resources/assets/rebornstorage/lang/de_de.lang:4]: Yes, the word is fine, but you might want to put a hyphen there.

Comment (ghost) [reply src/main/resources/assets/rebornstorage/lang/de_de.lang:4]: The refined storage translation didn't use a hyphen too. For example: "1k Speicherzelle".

File: src/main/resources/assets/rebornstorage/lang/de_de.lang (hunk 1 above)
Comment (ghost) [reply src/main/resources/assets/rebornstorage/lang/de_de.lang]: Hmm... what is about "Gehäuse"? It would be better I think.

**Comment by ghost** [comment]
@Vexatos ?

**Review by Vexatos — APPROVED** [review]

**Comment by modmuss50** [comment]
Thanks for sorting this out.

**Pull request merged**